### PR TITLE
XD68: add 65_ansi_split_bs layuout

### PR DESCRIPTION
**Pull request by owestend**
XD68 support also a split backspace.

## Description

What I did:
- add macro ```LAYOUT_65_ansi_split_bs``` to ```xd68.h```
- add this layout also to ```info.json```
- add this layout also to ```LAYOUTS``` in ```rules.mk```

What I **not** did:
- I did **not** add the additional key to ```LAYOUT_all```
- I did **not** change any other layout

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd68/info.json [hunk 1]
```diff
@@ -78,6 +78,80 @@
                 {"label":"Right", "x":15, "y":4}
             ]
         },
+        "LAYOUT_65_ansi_split_bs": {
+            "key_count": 68,
```
Comment (fauxpark) [inline keyboards/xd68/info.json:82]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/xd68/info.json [hunk 2]
```diff
@@ -78,6 +78,80 @@
                 {"label":"Right", "x":15, "y":4}
             ]
         },
+        "LAYOUT_65_ansi_split_bs": {
+            "key_count": 69,
```
Comment (fauxpark) [inline keyboards/xd68/info.json:82]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by owestend** [comment]
What else can I do to get these changes accepted?

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
> What else can I do to get these changes accepted?

In my case, I was busy with non-QMK stuff and I fell behind on my PR reviews. Sorry for that.

But looks good, thanks!

**Comment by owestend** [comment]
Thanks!

**Pull request merged**